book/src/SUMMARY.md

```diff
@@ -9,20 +9,20 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 - [Introduction](./index.md)
 - [Getting Started](./getting-started/index.md)
-    - [QObjects in Rust](./getting-started/1-qobjects-in-rust.md)
-    - [Our first CXX-Qt module](./getting-started/2-our-first-cxx-qt-module.md)
-    - [Creating the QML GUI](./getting-started/3-qml-gui.md)
-    - [Building with Cargo](./getting-started/4-cargo-executable.md)
-    - [Building with CMake](./getting-started/5-cmake-integration.md)
+  - [`QObject`s in Rust](./getting-started/1-qobjects-in-rust.md)
+  - [Our first CXX-Qt module](./getting-started/2-our-first-cxx-qt-module.md)
+  - [Creating the QML GUI](./getting-started/3-qml-gui.md)
+  - [Building with Cargo](./getting-started/4-cargo-executable.md)
+  - [Building with CMake](./getting-started/5-cmake-integration.md)
 - [Core Concepts](./concepts/index.md)
-    - [Build Systems](./concepts/build_systems.md)
```

Comment:
what is markdown lint and can we run it in CI ?

Reply:
from the [VSCode plugin page](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) 
>Linting is performed by the [markdownlint-cli2 engine](https://github.com/DavidAnson/markdownlint-cli2), which can be used in conjunction with this extension to provide command-line support for scripts and continuous integration scenarios. The [markdownlint-cli2-action GitHub Action](https://github.com/marketplace/actions/markdownlint-cli2-action) uses the same engine and can be integrated with project workflows.

Reply:
I've opened an issue for this. We can always add that to CI later :)